cli/lib/otel.ts

```diff
@@ -0,0 +1,56 @@
+import { trace } from '@opentelemetry/api'
+import { CollectorTraceExporter } from '@opentelemetry/exporter-collector'
+import { HttpInstrumentation } from '@opentelemetry/instrumentation-http'
+import { WinstonInstrumentation } from '@opentelemetry/instrumentation-winston'
+import { Resource, envDetector, processDetector } from '@opentelemetry/resources'
+import { NodeSDK } from '@opentelemetry/sdk-node'
+import { ResourceAttributes } from '@opentelemetry/semantic-conventions'
+import { BatchSpanProcessor } from '@opentelemetry/tracing'
+
+export const publishTracer = trace.getTracer('cube-publish')
```

Comment:
I'm not 100% sure if creating the tracer before the SDK was initialized works as expected.
What I've been doing in barnard59 is have a separate file with just
```ts
import { trace } from '@opentelemetry/api'

export const tracer = trace.getTracer('@cube-creator/cli')
```